### PR TITLE
Fixed the Helm parameter to set the service account annotations #1355

### PR DESCRIPTION
**Pull request by christianhxc**
**1. Issue, if available:**
When installing the Helm chart, it throws an error  #1355

**2. Description of changes:**
Added quotes for the annotation parameters and a missing backslash at the end of the command.

**3. How was this change tested?**
Manually tested by deploying the helm chart in my cluster.

**4. Does this change impact docs?**
- [x] Yes, PR includes docs updates
- [x] Yes, issue opened: #1355
- [ ] No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: website/content/en/v0.6.2/getting-started/_index.md [hunk 1]
```diff
@@ -156,7 +156,7 @@ Install the chart passing in the cluster details and the Karpenter role ARN.
 helm upgrade --install --namespace karpenter --create-namespace \
   karpenter karpenter/karpenter \
   --version v0.6.2 \
-  --set serviceAccount.annotations.eks\.amazonaws\.com/role-arn=${KARPENTER_IAM_ROLE_ARN}
+  --set serviceAccount.annotations."eks\.amazonaws\.com/role-arn"=${KARPENTER_IAM_ROLE_ARN} \
```
Comment (suket22) [inline website/content/en/v0.6.2/getting-started/_index.md:159]: So we generally don't update the versioned doc directly. Let me have someone else confirm

Comment (christianhxc) [reply website/content/en/v0.6.2/getting-started/_index.md:159]: @suket22 I see, this where the "Edit this page" link from the docs site redirected me. If I need to change it somewhere else instead, let me know :)

Comment (suket22) [reply website/content/en/v0.6.2/getting-started/_index.md:159]: I think you'd also want to update the preview docs too - https://github.com/aws/karpenter/tree/main/website/content/en/preview

**Review by suket22 — APPROVED** [review]

File: website/content/en/v0.6.2/getting-started/_index.md (hunk 1 above)
Comment (chrisnegus) [reply website/content/en/v0.6.2/getting-started/_index.md:159]: I agree that the changes should go in both places. Going forward, I think we should always fix the latest docs if they are broken and make sure the change goes into preview as well.

**Pull request merged**